tests/phpunit/tests/blocks/editor.php

```diff
@@ -568,6 +568,41 @@ public function test_block_editor_rest_api_preload_adds_missing_leading_slash( a
 		$this->assertStringContainsString( $expected, $haystack );
 	}
 
+	/**
+	 * @ticket 57547
+	 */
```

Comment:
```suggestion
	/**
	 * @ticket 57547
	 *
	 * @covers ::get_classic_theme_supports_block_editor_settings
	 */
```

Add @covers annotation.